examples/internal/proto/examplepb/echo_service.proto

```diff
@@ -90,4 +90,10 @@ service EchoService {
 			body: "body"
 		};
 	}
+	// EchoUnauthorized method receives a simple message and returns it.
```

Comment:
Could we update this comment to explain the purpose more clearly?
```suggestion
	// EchoUnauthorized method receives a simple message and returns it. It must
	// always return a google.rpc.Code of `UNAUTHENTICATED` and a HTTP Status code
	// of 401.
```

Reply:
I understand, thank you.